packages/cli/src/lib/implementation/only-plugins-utils.ts

```diff
@@ -42,21 +46,25 @@ export function filterCategoryByOnlyPluginsOption(
 
 export function validateOnlyPluginsOption(
   plugins: CoreConfig['plugins'],
-  { onlyPlugins }: { onlyPlugins?: string[] },
+  {
+    onlyPlugins,
+    verbose = false,
+  }: { onlyPlugins?: string[]; verbose?: boolean },
 ): void {
   const missingPlugins = onlyPlugins?.length
     ? onlyPlugins.filter(plugin => !plugins.some(({ slug }) => slug === plugin))
     : [];
 
   if (missingPlugins.length > 0) {
-    console.warn(
-      `${chalk.yellow(
-        '⚠',
-      )} The --onlyPlugin argument references plugins with "${missingPlugins.join(
-        '", "',
-      )}" slugs, but no such plugins are present in the configuration. Expected one of the following plugin slugs: "${plugins
-        .map(({ slug }) => slug)
-        .join('", "')}".`,
-    );
+    verbose &&
+      console.warn(
+        `${chalk.yellow(
+          '⚠',
+        )} The --onlyPlugin argument references plugins with "${missingPlugins.join(
+          '", "',
+        )}" slugs, but no such plugins are present in the configuration. Expected one of the following plugin slugs: "${plugins
+          .map(({ slug }) => slug)
+          .join('", "')}".`,
+      );
   }
```

Comment:
Same as previous comment:

```ts
if (missingPlugins.length > 0 && verbose) {
  console.warn(...);
}
```

Reply:
done. That was a nice small improvement. Sad i missed it.